projects/ion/src/lib/input-contador/input-contador.component.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { async } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { CommonModule } from '@angular/common';
+import { IonIconComponent } from './../icon/icon.component';
+import { render, fireEvent, screen } from '@testing-library/angular';
+import { InputContadorComponent } from './input-contador.component';
+
+const sut = async (customProps = {}) => {
```

Comment:
Missing return type on function .
```suggestion
const sut = async (customProps = {}): Promise<void> => {
```